
# problems with doing an API call

I want to do an API call using the Forecast air pollution data from this website
https://openweathermap.org/api/air-pollution
The API call of the website state that it is these:
http://api.openweathermap.org/data/2.5/air_pollution/forecast?lat={lat}&lon={lon}&appid={API key}

Now I have a dataframe in pandas which has the longitude  and latitude of 180 cities around the world.
To collect the forecast data I gave this
lon= df.Longitude
lat= df.Latitude
appid= 'b0gs3g26768234d11ss6jh722ff100r8e'
url = 'http://api.openweathermap.org/data/2.5/air_pollution/forecast?lat={lat}&lon={lon}&appid={b0gs3g26768234d11ss6jh722ff100r8e}'
r= requests.get(url)
r
r.text

But it always say invalid API key and my API key is activated. I don't know what I doing wrong. Could someone please help me

        